bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/XmlSequence.java

```diff
@@ -393,6 +393,18 @@ public XmlValue descendants(List<String> qnames) {
         return new XmlSequence(descendants);
     }
 
+    public XmlValue descendants() {
+        List<BXml> descendants = new ArrayList<>();
+        for (BXml child : children) {
+            if (child.getNodeType() == XmlNodeType.ELEMENT) {
+                XmlItem element = (XmlItem) child;
+                descendants.add(element);
+                addDescendants(descendants, element);
+            }
+        }
+        return new XmlSequence(descendants);
+    }
```

Comment:
Consider following:
```ballerina
    xml x = xml `<a><b>helo</b></a>`;
    xml b = x/*;
    if (b is xml:Element) {
        xml c = b.getDescendants();
    }
 ```

`c` should only contain `hello`, at the moment it's `<b>helo</b>helo`

Reply:
`b` is a sequence with a single element, hence it's a xml:Element.
calling `getDescendants` on a xml:Element should only return children of that element.

Reply:
Handled with a4dbd75508baab046999c18b524530019c699be9